Fix cobra

marouenbg opened the pull request:
@Enakshi-Saha Please have a look!

marouenbg committed:
fix rm missing

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
Merge remote-tracking branch 'upstream/devel' into devel

marouenbg committed:
cobra hotfix

marouenbg committed:
cobra hotfix

marouenbg committed:
cobra hotfix

marouenbg committed:
cobra hotfix

marouenbg committed:
add transpose of psi

katehoffshutta commented:
@Enakshi-Saha @marouenbg, we have a release upcoming for netZooPy and I am wondering if we should incorporate this PR. If you could provide background and updates on the above, that would be helpful!

marouenbg commented:
Hey @katehoffshutta , let's keep it hanging at the moment, I am working on some mods to this PR and then I will send it your way. Enakshi found a way to improve COBRA and I am testing it now

marouenbg committed:
Fix assignment operator for Lasso model

marouenbg committed:
Merge branch 'devel' into fix-cobra

marouenbg committed:
Refine parameter description and update model fitting

Updated the design matrix parameter description and modified model fitting to exclude intercept.

marouenbg committed:
Fix pandas indexing in cobra test: convert X to numpy before mean

marouenbg committed:
Fix pandas/numpy compat in condor: use iloc and np.asarray for sum

marouenbg committed:
Rename cobra 'deprecated' mode to 'MLE' and populate alpha param docs

marouenbg committed:
Add mode parameter to cobra: 'corr' (default) or 'cov' for covariance decomposition

marouenbg commented:
@katehoffshutta this one is tip-top and ready to go!

marouenbg committed:
Merge devel into fix-cobra, resolve condor conflicts

katehoffshutta committed:
Merge branch 'devel' into fix-cobra

katehoffshutta commented:
@marouenbg, can you please check out the error on build for ubuntu?https://github.com/netZoo/netZooPy/actions/runs/24408340647/job/71297828492?pr=367#logs

marouenbg committed:
Retrigger CI (transient SSL error on ubuntu 3.9 runner)

marouenbg commented:
@katehoffshutta it is good to go!
